varanny.go

```diff
@@ -247,7 +255,7 @@ func handleConnection(conn net.Conn, p *program) {
 
 		close(dbfsLevels)
 		close(cmdChannel)
-		//		close(stop)
+		close(stop)
```

Comment:
Note: I uncommented this. I know it doesn't "do" anything useful but it felt cleaner should the stop command ever want to be extended to do something.